.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
```suggestion
18
```

I can’t see any dependencies here that would be incompatible with 18.

Reply:
Thanks, updated.